larq_compute_engine/mlir/python/end2end_test.py

```diff
@@ -0,0 +1,70 @@
+import sys
+import pytest
+import larq as lq
+import subprocess
+import tensorflow as tf
+import numpy as np
+
+from larq_compute_engine.mlir.python.converter import convert_keras_model
+
+
+@pytest.fixture
+def model():
+    img_input = tf.keras.layers.Input(shape=(4, 4, 1))
+    # A layer with zero-padding
+    out = lq.layers.QuantConv2D(
+        filters=32,
+        kernel_size=3,
+        padding="same",
+        input_quantizer="ste_sign",
+        kernel_quantizer="ste_sign",
+        use_bias=False,
+        activation="relu",
+    )(img_input)
+    out = tf.keras.layers.BatchNormalization(
+        gamma_initializer=tf.keras.initializers.RandomNormal(1.0),
+        beta_initializer="uniform",
+    )(out)
+    # A layer with one-padding
+    out = lq.layers.QuantConv2D(
+        filters=16,
+        kernel_size=3,
+        padding="same",
+        # pad_values=1.0,
+        input_quantizer="ste_sign",
+        kernel_quantizer="ste_sign",
+        use_bias=False,
+        activation="relu",
+    )(out)
+    out = tf.keras.layers.BatchNormalization(
+        gamma_initializer=tf.keras.initializers.RandomNormal(1.0),
+        beta_initializer="uniform",
+    )(out)
+    out = tf.keras.layers.GlobalAvgPool2D()(out)
+    return tf.keras.Model(inputs=img_input, outputs=out)
+
+
+def test_simple_model(model, tmp_path):
+    converted = convert_keras_model(model)
+    modelfile = tmp_path / "end2end_model.tflite"
+    with modelfile.open("wb") as f:
+        f.write(converted)
+
+    x = np.zeros(shape=(1, 4, 4, 1))
+    y = model(x)
+
+    # The model emits 16 floats. Write them to a text file
+    y = np.squeeze(y)
+
+    expected_output_file = tmp_path / "expected_output.txt"
+    with expected_output_file.open("w") as f:
+        np.savetxt(f, y, delimiter=" ")
```

Comment:
This should be enough:
```suggestion
    np.savetxt(expected_output_file, y, delimiter=" ")
```

Reply:
I'll do that together with the random-input commit.